pkg/controller/globalscylladbmanager/sync_scylladbmanagerclusterregistrations.go

```diff
@@ -38,6 +44,14 @@ func (gsmc *Controller) syncScyllaDBManagerClusterRegistrations(ctx context.Cont
 		}
 
 		maps.Copy(requiredScyllaDBManagerClusterRegistrations, requiredScyllaDBManagerClusterRegistrationsForScyllaDBDatacenters)
+
+		var requiredScyllaDBManagerClusterRegistrationsForScyllaDBClusters map[string][]*scyllav1alpha1.ScyllaDBManagerClusterRegistration
+		requiredScyllaDBManagerClusterRegistrationsForScyllaDBClusters, err = makeScyllaDBManagerClusterRegistrationsForScyllaDBClusters(scyllaDBClusters)
+		if err != nil {
+			return fmt.Errorf("can't make required ScyllaDBManagerClusterRegistration objects for ScyllaDBClusters(s): %w", err)
+		}
+
+		maps.Copy(requiredScyllaDBManagerClusterRegistrations, requiredScyllaDBManagerClusterRegistrationsForScyllaDBClusters)
```

Comment:
it's a good time to move both these make's to single `makeScyllaDBManagerClusterRegistrations`

Reply:
done